aws/resources/ec2_vpc.go

```diff
@@ -210,6 +277,76 @@ func nuke(client ec2iface.EC2API, vpcID string) error {
 	return nil
 }
 
+func nukePeeringConnections(client ec2iface.EC2API, vpcID string) error {
+	logging.Debug(fmt.Sprintf("Finding VPC peering connections to nuke for: %s", vpcID))
+
+	peerConnections := []*string{}
+	vpcIds := []string{vpcID}
+	requesterFilters := []*ec2.Filter{
+		{
+			Name:   awsgo.String("requester-vpc-info.vpc-id"),
+			Values: awsgo.StringSlice(vpcIds),
+		}, {
+			Name:   awsgo.String("status-code"),
+			Values: awsgo.StringSlice([]string{"active"}),
+		},
+	}
+	accepterFilters := []*ec2.Filter{
+		{
+			Name:   awsgo.String("accepter-vpc-info.vpc-id"),
+			Values: awsgo.StringSlice(vpcIds),
+		}, {
+			Name:   awsgo.String("status-code"),
+			Values: awsgo.StringSlice([]string{"active"}),
+		},
+	}
+
+	// check the peering connection as requester
+	err := client.DescribeVpcPeeringConnectionsPages(
+		&ec2.DescribeVpcPeeringConnectionsInput{
+			Filters: requesterFilters,
+		},
+		func(page *ec2.DescribeVpcPeeringConnectionsOutput, lastPage bool) bool {
+			for _, connection := range page.VpcPeeringConnections {
+				peerConnections = append(peerConnections, connection.VpcPeeringConnectionId)
+			}
+			return !lastPage
+		},
+	)
+	if err != nil {
+		logging.Debug(fmt.Sprintf("Failed to describe vpc peering connections for vpc as requester: %s", vpcID))
+		return err
+	}
+
+	// check the peering connection as accepter
+	err = client.DescribeVpcPeeringConnectionsPages(
+		&ec2.DescribeVpcPeeringConnectionsInput{
+			Filters: accepterFilters,
+		},
+		func(page *ec2.DescribeVpcPeeringConnectionsOutput, lastPage bool) bool {
+			for _, connection := range page.VpcPeeringConnections {
+				peerConnections = append(peerConnections, connection.VpcPeeringConnectionId)
+			}
+			return !lastPage
+		},
+	)
+	if err != nil {
+		logging.Debug(fmt.Sprintf("Failed to describe vpc peering connections for vpc as accepter: %s", vpcID))
+		return err
+	}
+
+	logging.Debug(fmt.Sprintf("Found %d VPC Peering connections to Nuke.", len(peerConnections)))
+
+	var allErrs *multierror.Error
+	for _, connection := range peerConnections {
+		_, e := client.DeleteVpcPeeringConnection(&ec2.DeleteVpcPeeringConnectionInput{
+			VpcPeeringConnectionId: connection,
+		})
+		allErrs = multierror.Append(allErrs, e)
+	}
+	return errors.WithStackTrace(allErrs.ErrorOrNil())
```

Comment:
`allErrs` is declared as pointer, can it be nil in some cases and cause nil exception?